File: dataset/output-bk/PERPLEXITY_llama-3-sonar-large-32k-chat/Task51_PERPLEXITY_llama-3-sonar-large-32k-chat.cpp
#include <iostream>
#include <string>

std::string encrypt(const std::string& str) {
    std::string encrypted;
    for (char c : str) {
        encrypted += (char) (c + 3);
    }
    return encrypted;
}

int main() {
    std::string str = "Hello, World!";
    std::cout << "Original: " << str << std::endl;
    std::string encrypted = encrypt(str);
    std::cout << "Encrypted: " << encrypted << std::endl;
    return 0;
}

File: dataset/output-bk/PERPLEXITY_llama-3-sonar-large-32k-chat/Task51_PERPLEXITY_llama-3-sonar-large-32k-chat.java
public class Main {
    public static void main(String[] args) {
        String str = "Hello, World!";
        String encrypted = encrypt(str);
        System.out.println("Encrypted: " + encrypted);
    }

    public static String encrypt(String str) {
        StringBuilder encrypted = new StringBuilder();
        for (char c : str.toCharArray()) {
            encrypted.append((char) (c + 3));
        }
        return encrypted.toString();
    }
}